### PR TITLE
Recommend UUID for deviceId and groupId

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

File: getusermedia.html [hunk 1]
```diff
@@ -2944,6 +2944,14 @@ <h2>Methods</h2>
                           "MediaDeviceInfo">groupId</a></code> member be a
                           newly generated unique identifier.</p>
                         </li>
+                        <p>A good practice for generating <code><a
+                          data-link-for="MediaDeviceInfo">deviceId</a></code>
+                          and <code><a data-link-for="MediaDeviceInfo">groupId</a></code>
+                          values is to use a UUID [[rfc4122]], which is 36
+                          characters long in its canonical form. To avoid
+                          fingerprinting, implementations SHOULD use the forms in
+                          section 4.4 or 4.5 of RFC 4122.
```
Comment (alvestrand) [inline getusermedia.html]: This is the CL that should take out the corresponding language in the "id" attribute of 4.2 "MediaStream".

Comment (youennf) [reply getusermedia.html]: I am not sure to understand what you suggest.
If we do not want this text in two different places, we can move it to a single place and link to it from MediaStream.id, MediaDeviceInfo.deviceId and MediaDeviceInfo.groupId

File: getusermedia.html [hunk 2]
```diff
@@ -2944,6 +2944,14 @@ <h2>Methods</h2>
...
+                        </p>
```
Comment (alvestrand) [inline getusermedia.html]: Missing ```</li> ```

**Comment by alvestrand** [comment]
Merge once Travis goes green.

**Comment by youennf** [comment]
Rebased and moved the p section inside groupId li section.

**Pull request merged**